packages/runtime/runtime-utils/src/index.ts

```diff
@@ -10,8 +10,7 @@ export * from "./objectstoragepartition";
 export * from "./objectstorageutils";
 export * from "./requestParser";
 export * from "./serializer";
-export * from "./summarizerNode";
-export * from "./summarizerNodeWithGc";
```

Comment:
I wish there were a more explicit indication of what files are _not_ exported via index.  Oh well.

Reply:
made a comment on summarizerNodeContract that i think might help here

---
In reply to: [529063164](https://github.com/microsoft/FluidFramework/pull/4447#discussion_r529063164) [](ancestors = 529063164)